src/__tests__/mocks/testConfiguration.ts

```diff
@@ -0,0 +1,13 @@
+import { IFusionAuthConfiguration } from '../../providers/FusionAuthProvider';
+
+export const TEST_REDIRECT_URL =
+    'http://localhost?code=here&state=00000000000000000000000000000000000000000000000000000000&error=none';
+
+export const TEST_CONFIGURATION: IFusionAuthConfiguration = {
```

Comment:
Can we rename to be a little more succinct?

```suggestion
export const TEST_CONFIG: IFusionAuthConfig = {
```